src/components/FormButton.vue

```diff
@@ -2,10 +2,10 @@
   <button :disabled="disabled"
     class="button"
     :class="[variant, display, icon ? 'icon' : '']">
```

Comment:
`display` is not being set anywhere

```suggestion
    :class="[variant, icon ? 'icon' : '']">
```